crates/commitlog/src/repo/fs.rs

```diff
@@ -0,0 +1,108 @@
+use std::io;
+use std::{
+    fs::{self, File},
+    path::PathBuf,
+};
+
+use log::debug;
+
+use super::Repo;
+
+const SEGMENT_FILE_EXT: &str = ".stdb.log";
+
```

Comment:
`.log` extension could be confused easily as being text-based, how about `.log.stdb` or `.stdb.wbl` (write behind log)

Reply:
This file extension was requested by @Centril. If we don’t like it, an amendment to the spec shall be proposed.

Reply:
.log is pretty standard. I think it's the same for kafka IIRC.